### ISSUE TITLE
No posts

### ISSUE THREAD

**Issue by Ishan-pro** [Ishan-pro/philosophyblog#3]
There should be plenty of posts in a blog but there isn't a post in this blog. A post needs to have a
- Title
- Tag
- Content in markdown and rendered in html
- Date of publishing
- Comments by authorized users

**Comment by LeafyOjas** [comment]
Hi , I would like to work on this issue . Is there a need to add styling , like any specific design?

**Comment by Ishan-pro** [comment]
> Hi , I would like to work on this issue . Is there a need to add styling , like any specific design?

Yes, there is we plan to use tailwindcss, react-router, redux-toolkit and react-markdown to use in posts. 

There should be two pages one with the facility to list blogposts and the other with the facility to view them in detail. 

We wish to use supabase in the backend.

**Comment by LeafyOjas** [comment]
Okay , that sounds good , I am new to tailwind-css so have to see for that.  But for this issue I guess there is fake data generated or some hardcoded post-data will be used.
